float64 latitude
float64 longitude
float64 elevation
float32 roll
float32 pitch
float32 heading
int32 gear